webpack/global_index.js

```diff
@@ -50,7 +53,8 @@ addGlobalFill('host-overview-cards', 'Installable errata', <ErrataOverviewCard k
 // Details tab cards & card extensions
 addGlobalFill('host-tab-details-cards', 'Installed products', <InstalledProductsCard key="installed-products" />, 100);
 addGlobalFill('host-tab-details-cards', 'Registration details', <RegistrationCard key="registration-details" />, 200);
-addGlobalFill('host-details-tab-properties-1', 'Subscription UUID', <SystemPropertiesCardExtensions key="subscription-uuid" />);
+addGlobalFill('host-details-tab-properties-subscription', 'Subscription UUID', <SystemPropertiesCardSubscription key="subscription-uuid" />);
+addGlobalFill('host-details-tab-properties-tracer', 'Tracer', <SystemPropertiesCardTracer key="tracer-status" />);
```

Comment:
But I guess it's more descriptive, so :+1:

Reply:
but then we need to add these slots to foreman instead of using the existing one

Reply:
Well, the PR in Foreman adds a new slot anyway, it just renames the old one as a side effect.

Reply:
Yeah, I don't think we need to slots there as this works fine using one slot. Also we shouldn't make slot names for specific plugins as they meant to be used generally.

Reply:
> no need to change the name, the id `host-details-tab-properties-1` can be used for both components :)

@MariaAga Mockup displays these two components in different positions in the card. How do you use one id for both?

Reply:
> Could you explain what the concern is here please?
I want to avoid adding extra slots if not needed, but if design wise they should be in different places then it makes sense to add another slot.

Reply:
How about `host-details-tab-properties-tracer-installed` and `host-details-tab-properties-subscription-uuid`?